doc/Administration-guide/Tenants-and-applications-management/index.md

```diff
@@ -10,83 +10,75 @@ sort_idx: 40
+This user can create, edit, and delete tenant administrators.
+To log in to the Administration UI as a Kaa administrator, use the username and password previously created for the Kaa administrator.
 
 ![](attach/image2014-12-22%2016-22-58.png)
 
-To customize the account, click **Settings => Profile** in the upper right corner of the window and change the first/last name and e-mail to the private ones. Click **Save** to apply the changes.
-Note that **Email** is a mandatory field.
-
-<img src="attach/image2014-11-12%2013-3-23.png" width="600" height="400">
-
-To set a private password, click **Settings => Change password** and fill in the fields as required.
-
-![](attach/image2014-11-10%2017-31-31.png)
-
-## Managing Tenants
+To customize the account:
 
-The Kaa IoT platform supports Multi-tenant architecture. It allows _Kaa admin_ to create separate scope for each instance of Tenant.
-Unlike _Kaa admin_, entities _Tenant Admin_ and _Tenant Developer_ can be visible only in the scope of single Tenant.
-For more information about Multitenancy, check related description on [wiki]( https://en.wikipedia.org/wiki/Multitenancy ).
+1. Click **Settings** > **Profile**.
 
-# Tenant admin
+2. On the **Account profile** page, enter your first name, last name and email address.
+	![](attach/image2014-11-12%2013-3-23.png)
 
-The _tenant admin_ is a Kaa user who is responsible for managing applications, users and event class families.
-To log into the Admin UI as a tenant admin, use the previously generated username/password for the tenant admin.
-To customize the account, click **Settings => Profile** and change the first/last name and e-mail to private ones.
-To set a private password, click **Settings => Change password** and fill in the fields as required.
+3. If you want to change password, click the **Change password** button.
+	![](attach/image2014-11-10%2017-31-31.png)
 
-## Managing applications
+4. Click **Save** to save the account profile changes.
 
-As a tenant admin, you can add and edit applications.
+Kaa platform supports multi-tenant architecture.
+It allows Kaa administrator to create separate scopes for each instance of a tenant.
+Unlike Kaa administrator, tenant administrator and tenant developer are only visible in the scope of a single tenant.
+For more information about multitenancy, see [Wikipedia](https://en.wikipedia.org/wiki/Multitenancy).
 
-To create a new application, do the following:
+## Tenant administrator
 
-1. Open the **Applications** window by clicking the corresponding link on the navigation panel.
+Tenant administrator is a Kaa user that manages applications, users, and [event class families]({{root_url}}Programming-guide/Key-platform-features/Events/#event-class-families).
 
-    <img src="attach/image2015-3-4%2016-47-22.png" width="400" height="250">
+To log in to the Administration UI as a tenant administrator, use the previously generated username and password.
```

Comment:
add   
`(or use [default ones]({{root_url}}Getting-started/#administration-ui)) `
at the end of line
because process  of creation is not described on this page